source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,304 @@
+.. _swupd-3rd-party:
+
+swupd 3rd-party 
+###############
+
+Upstream |CL| offers a plethora of `bundles`_ to choose from.  
+
+To provide you access to even more software outside of the distro, 
+the :ref:`mixer` and :ref:`swupd-guide` tools have added support for 3rd-party
+bundles starting with release 32570.  
```

Comment:
End sentence at "..bundles" (thereby removing "starting with..."

Moving the specific release, 32570,  when 3rd party was made available to Prereqs caters to readers who scan.

Reply:
Thanks. Looks great.